infrastructure.md

```diff
@@ -127,3 +140,7 @@ send email.  Currently this includes:
 
 
 [Azure portal]: https://portal.azure.com
+[sqitch]: https://sqitch.org/
+[password file]: https://www.postgresql.org/docs/10/libpq-pgpass.html
+[connection service file]: https://www.postgresql.org/docs/10/libpq-pgservice.html
+[the connection service file used for Metabase]: https://github.com/seattleflu/backoffice/blob/master/pg_service.conf
```

Comment:
Wording is different than line 99-100, so the link doesn't work.

Reply:
Thank you for catching that.